src/model/control_desk.py

```diff
@@ -284,28 +311,32 @@ def unlink(self) -> bool:
             else:
                 i += 1
         BankSet._fish_connector.send_fader_bank_set_delete_message(self.id)
-        BankSet._linked_bank_sets.pop(found_index)
+        BankSet._linked_bank_sets.pop(found_index)  # TODO found index koente leer sein
         return True
 
     @property
     def is_linked(self) -> bool:
+        """TODO docstring"""
         return self.pushed_to_fish
 
 
 def set_network_manager(network_manager: NetworkManager):
+    """TODO docstring"""
     BankSet._fish_connector = network_manager
 
 
 def set_seven_seg_display_content(content: str):
+    """TODO docstring"""
```

Comment:
"""Set the content of the 7seg displays of connected X-Touch controllers.

Fish will truncate any text longer than 12 chars and only guarantees the correct display of hexadecimal characters due to the limitations of seven segment displays.
"""